src/trainable/DiscountedCfrTrainable.cpp

```diff
@@ -185,7 +185,7 @@ json DiscountedCfrTrainable::dump_evs() {
     json retjson;
     retjson["actions"] = std::move(actions_str);
     retjson["evs"] = std::move(evs);
-    return std::move(retjson);
+    return retjson;
```

Comment:
This went in here by incident. It was a suggestion from the Qt-Creator-GUI, that the sdt:move would "prevent copy elision", and that was offered as a fix, which I apparently accidentally applied.

Reply:
Not sure why I wrote this, did you compair the dumping speed before you fix it and after?

Reply:
I did not look at the dumping speed separately, the overall solving process was not noticeably different. I was quite surprised to the the Json library being used to collect strategies and EVs by the solver -- is that the Java legacy?

Reply:
> I did not look at the dumping speed separately, the overall solving process was not noticeably different. I was quite surprised to the the Json library being used to collect strategies and EVs by the solver -- is that the Java legacy?

yes, totally

My plan was to rewrite the entire dumping thing last winter, but I get lazy LOL